roles/infinispan/tasks/install.yml

```diff
@@ -128,10 +133,10 @@
   register: path_to_workdir
   become: yes
 
-- name: "Create target directory {{ infinispan.installation_path | dirname | dirname }}"
+- name: "Create target directory {{ infinispan.installation_path | dirname }}"
```

Comment:
are you sure about this? as far as i can see it should go dir up twice

Reply:
To be fair, I've only tested this with JDG given the following configuration:

```yaml
jdg:
  installation:
    base_path: "/opt/jdg"
    version: "8.4.2"
[..]
jdg_install_dir: "{{ jdg.installation.base_path }}/redhat-datagrid-{{ jdg.installation.version }}-server"
[..]
infinispan_installation_path: "{{ jdg_install_dir }}"
```

And here said lines escalated to `/opt` (changing its permissons to `infinispan` user accordingly)

The role uses the following - *similar* - default values for ispan...

```yaml
infinispan_dest: /opt/infinispan
infinispan_installation_path: "{{ infinispan_dest }}/infinispan-server-{{ infinispan_version }}/"
```

... I think we only want to go up once since we don't want to mess with `/opt`

Reply:
I see your point; I'll need to test the change in the downstream CI (where we test the certified collection). Btw, did you know there are certified collection (on automation hub, not galaxy) that target specifically the red hat products?

Reply:
I was not aware about the atuomation hub, but thought that there's more to these kind of collection, given the `downstream` variables ;-). Thanks for the hint, will have a look.